indexFormat/implementation.md

```diff
@@ -1,77 +1,67 @@
 
-In this guide, we cover how an LSIF implementation can be used in the context of Rich Code Navigation.
+## The Rich Code Navigation scenario
 
-## Rich Code Navigation scenario
+With Rich Code Navigation, users use navigate features (peek definition, find all references, diagnostics, etc.) over PRs in their editor without having a local checkout. These navigation features are powered by a cloud language service, which uses an LSIF index. The index can be generated at a variety of places, like for instance in the CI pipeline:
 
-With Rich Code Navigation, users can use navigation features on their pull requests, without requiring a local checkout. In this scenario, the navigation features are powered through an external service. This service uses a pre-built LSIF index. The index can be generated at a variety of places, like for instance the CI pipeline of the repo:
+1. User creates a new PR.
+1. The CI configured on the repo builds the PR.
+1. The LSIF exporter runs on CI and generates the LSIF index.
 
-1. User creates a pull request
-1. The CI tool builds this pull request
-1. The LSIF exporter runs in the CI pipeline and generates the LSIF file
-
-## LSIF exporter implementations
+## LSIF exporters
 
 | Language | Repository |
 |--|--|
 | TypeScript/JavaScript | [lsif-node](https://github.com/Microsoft/lsif-node) |
+| Java | |
+| C# | |
 
-> Are we missing an implementation? Create a new issue on GitHub.
+> Are we missing an implementation? File a new issue on GitHub to add it here.
 
-## LSIF implementation skeleton
+## LSIF exporter skeleton
 
-As detailed in the specification, the LSIF exporter consists of two tools: the index exporter and the package manager linker.
+As [detailed in the spec](specification.md#project-exports-and-external-imports), the LSIF exporter consists of two tools: the index exporter and the package linker.
 
 ### Index exporter
 
 This tool generates an LSIF dump for a workspace by traversing through source files and storing LSP responses. For TypeScript/JavaScript, [`lsif-tsc`](https://github.com/Microsoft/lsif-node/tree/master/tsc) is the index exporter.
 
-### Package manager linker
+### Package linker
 
-This tool ingests the LSIF output from an index exporter and converts `moniker` vertices into the package manager scheme. Additionally, it links these monikers to `packageInformation` vertices. For TypeScript/JavaScript, [`lsif-npm`](https://github.com/Microsoft/lsif-node/tree/master/npm) is the package manager linker for NPM.
+This tool ingests the LSIF output from an index exporter and converts `moniker` vertices into the package manager scheme. Additionally, it links these monikers to `packageInformation` vertices that reference packages hosted on a registry. For TypeScript/JavaScript, [`lsif-npm`](https://github.com/Microsoft/lsif-node/tree/master/npm) is the package manager linker for NPM.
```

Comment:
Reading the document up to this point, we haven't been introduced to the concept of a package manager scheme. Can you link to where that's defined or define it here?